ui/packages/atoms/src/IconButton.svelte

```diff
@@ -6,6 +6,7 @@
 <button
 	on:click
 	class="text-gray-500  bg-white/90 h-5 w-5 flex items-center justify-center rounded shadow-sm hover:shadow-xl hover:ring-1 ring-inset ring-gray-200 z-10 dark:bg-gray-900 dark:ring-gray-600"
+	{...$$restProps}
```

Comment:
I'm not a huge fan of `$$restProps` because it is hard to work out what is being passed down and results in code that isn't as well optimised (by the Svelte compiler). It is useful when we don't know the props at build time, but that isn't the case here. 

Since we know these values ahead of time, we can just define a `label` prop in this component (`export let label`) and set that as the `aria-label` explicitly.